packages/x-date-pickers/src/PickersCalendarHeader/PickersCalendarHeader.tsx

```diff
@@ -189,6 +189,15 @@ type PickersCalendarHeaderComponent = (<TDate>(
   props: PickersCalendarHeaderProps<TDate> & React.RefAttributes<HTMLButtonElement>,
 ) => React.JSX.Element) & { propTypes?: any };
 
+/**
+ * Demos:
+ *
+ * - [Custom slots and subcomponents](https://mui.com/x/react-date-pickers/custom-components/)
```

Comment:
WDYT about also adding the `DateCalendar` and `DateRangeCalendar` pages as they showcase this component?
```suggestion
 * - [DateCalendar](https://mui.com/x/react-date-pickers/date-calendar/)
 * - [DateRangeCalendar](https://mui.com/x/react-date-pickers/date-range-calendar/)
 * - [Custom slots and subcomponents](https://mui.com/x/react-date-pickers/custom-components/)
```